test/SearchTest.php

```diff
@@ -294,6 +291,11 @@ public function testCallingNextAfterIterationShouldNotThrowException()
             // do nothing - just iterate
         }
         $items->next();
+        // The "pass" expectation here is just that no exception was thrown.
+        // Getting to this point in the code is the "definition of pass".
+        // phpunit will flag this as a risky test if we do not assert anything,
+        // so assert something.
+        $this->assertIsObject($items);
```

Comment:
Similar to the above: perhaps we should check what the `->current()` value is.

Reply:
In PR #13 I have done:
```
        $this->assertIsArray($items->current());
```